LML.NPOManagement.Bll/Services/UserService.cs

```diff
@@ -263,5 +263,44 @@ public async Task<bool> ModifyUserInfo(UserInformationModel userInformationModel
             await _dbContext.SaveChangesAsync();
             return true;
         }
+
```

Comment:
Vlad jan I would suggest clarifying comments in the method header, so the developers know what is username and userId. I also suggest changing the parameter names: userName -> searchedUserNameStartChars
userId -> loggedInUserId.
I think that way it becomes clear what parameters to pass to your business method